tests/kafkatest/services/kafka/kafka.py

```diff
@@ -463,6 +463,18 @@ def reconfigure_zk_for_migration(self, kraft_quorum):
         # This is not added to "advertised.listeners" because of configured_for_zk_migration=True
         self.port_mappings[kraft_quorum.controller_listener_names] = kraft_quorum.port_mappings.get(kraft_quorum.controller_listener_names)
 
+    def downgrade_kraft_broker_to_zk(self, kraft_quorum):
+        self.configured_for_zk_migration = True
+        self.quorum_info = quorum.ServiceQuorumInfo(quorum.zk, self)
+        self.controller_quorum = kraft_quorum
+
+        # Set the migration properties
+        self.server_prop_overrides.extend([
+            ["zookeeper.metadata.migration.enable", "true"],
+            ["controller.quorum.voters", kraft_quorum.controller_quorum_voters],
+            ["controller.listener.names", kraft_quorum.controller_listener_names]
+        ])
```

Comment:
In https://kafka.apache.org/documentation/#kraft_zk_migration under "Reverting to ZooKeeper mode During the Migration" we say:

> On each broker, remove the zookeeper.metadata.migration.enable, controller.listener.names, and controller.quorum.voters configurations.

And before this method is called we log `"Rolling brokers back to ZK mode"`.

I'm guessing I'm probably missing something, shouldn't these three properties be removed?